http-servers/src/test/java/com/palantir/remoting1/servers/DropwizardTracingFiltersTest.java

```diff
@@ -115,7 +115,7 @@ public void testLogAppenderCanAccessTraceId() throws Exception {
         @Override
         public void run(Configuration config, final Environment env) throws Exception {
             env.jersey().register(new TestEchoResource());
-            DropwizardTracingFilters.registerTracers(env, config, "testTracerName");
+            DropwizardServers.configure(env, config, "testTracerName", true);
```

Comment:
I'm pretty happy to test these things end-to-end for now. a test refactor is bound to happen soon though.